apps/api/lib/api_web/controllers/product_controller.ex

```diff
@@ -12,7 +12,7 @@ defmodule ApiWeb.ProductController do
     render(conn, "products.json", products: products)
   end
 
-  def product(conn, params) do
+  def show(conn, params) do
     product =
       Repo.get_by(Product, slug: params["product_slug"])
```

Comment:
As the router has been changed to resources product slug will be in `params["product_slug"]`